src/app/BoostTable.js

```diff
@@ -12,13 +12,18 @@ const BoostTable = ({
   showOldBoosts,
   target,
   artifactBoostBoostBonus,
+  dilithiumBoostBonus,
 }) => {
   const [limit] = useState(50)
   const [sortBy, setSortBy] = useState('cost')
 
   const boostsToShow = useMemo(() => {
     let combos = boostCombinations
 
+    if(dilithiumBoostBonus && dilithiumBoostBonus > 0) combos.forEach((c) => {
+      c.time = c.getComboTime() * (dilithiumBoostBonus);
```

Comment:
nit: are parenthesis needed? `(dilithiumBoostBonus)`

Reply:
Refactored entirely, actually - because I'm providing a default of 1 when initializing the boost table, it can get one-lined in this instance;

```
          <BoostTable
            target={targetChickenCount}
            hatchRate={hatchRate}
            hasProPermit={hasProPermit}
            showOldBoosts={showOldBoosts}
            artifactBoostBoostBonus={artifactBoostBoostBonus}
            dilithiumBoostBonus={dilithiumBoostBonus ? dilithiumBoostBonus : 1}
          />
```

so;

`combos.forEach(c => c.time = c.getComboTime() * dilithiumBoostBonus);`

This'll be coming with the commits soon :)